src/colorize.cr

```diff
@@ -253,37 +244,50 @@ module Colorize
       io << blue
     end
   end
+
+  # A text decoration.
+  #
+  # Note that not all text decorations are supported in all terminals.
+  # When a text decoration is not supported, it will leave the text unaffected.
+  @[Flags]
+  enum Mode
+    # Makes the text bold.
+    Bold = 1
+    # Makes the text color bright.
+    Bright = 1
+    # Dims the text color.
+    Dim
+    # Underlines the text.
+    Underline
+    # Makes the text blink slowly.
+    Blink
+    # Swaps the foreground and background colors of the text.
+    Reverse
+    # Makes the text invisible.
+    Hidden
+  end
+end
+
+private def each_code(mode : Colorize::Mode)
+  yield '1' if mode.bold?
```

Comment:
I know the result would be the same, but for readability I wish it was explicitly covering all enum members.
```cr
yield '1' if mode.bold? || mode.bright?
```

Maybe compiler could be (is) smart enough to make it noop, but humans would see all cases covered.
Or maybe just comment.

Reply:
I don't think the compiler can easily optimize this.

I'll add a comment to the member definition. That's probably more useful than here in this internal implementation.